builder/role_image.go

```diff
@@ -24,8 +24,9 @@ import (
 )
 
 const (
-	binPrefix             = "bin"
-	jobConfigSpecFilename = "config_spec"
+	binPrefix               = "bin"
+	jobConfigSpecFilename   = "config_spec"
+	roleCacheConfigBasename = "roleConfigs.json"
```

Comment:
This new const seems to be remnant of when the change used a separate cache file for hashes.
AFAICT it is not needed in the current code.